build.gradle.kts

```diff
@@ -21,6 +22,25 @@ allprojects {
     }
 }
 
+kotlin {
+    val xcframeworkName = "GodToolsSharedXCFramework"
+    val xcf = XCFramework(xcframeworkName)
+
+    listOf(
+        iosArm64(),
+        iosSimulatorArm64(),
+    ).forEach {
+        it.binaries.framework {
+            baseName = xcframeworkName
+
+            // Specify CFBundleIdentifier to uniquely identify the framework
+            binaryOption("bundleId", "org.cru.${xcframeworkName}")
+            xcf.add(this)
+            isStatic = true
+        }
+    }
+}
+
```

Comment:
I would like to move building the SPM module to a new `:publishing:spm` or `:publishing:swift` project to quit using the root project for it. This will be similar to how `:publishing:npm` is setup for publishing the js target